nikola/data/themes/base/templates/feeds_translations_helper.tmpl

```diff
@@ -1,91 +1,97 @@
 ## -*- coding: utf-8 -*-
 
-## Handles both feeds and translations
-<%def name="head(classification=None)">
-    % if rss_link:
+<%def name="_append_language(language)">${ " (" + language + ")" if len(translations) > 1 else "" }</%def>
```

Comment:
If it would be only a bit more, I'd agree. But here it will be a lot.

Except if I add a function for generating a `<link>` or `<a>` which contains all the cases in there and has some more additional variables. I don't know if that's easier to read.